src/components/useraccount/UserProfile.vue

```diff
@@ -4,7 +4,21 @@
     <div class="row">
       <div class="col-sm-3">
         <!--{{> myaccount/my-account-sidebar myPersonalDetailsTab=true}}-->
+        <div class="my-account-sidebar">
+          <div id="my-account-mobile-content"
+               class="my-account-sidebar-items active">
+            <span>Personal Details</span>
+          </div>
+          <div class="my-account-sidebar-items">
+            <button @click="logout">
+              <span>
+                {{ $t("myAccountSidebar.signOut") }}
+              </span>
+            </button>
+          </div>
+        </div>
```

Comment:
I know it's temporary, but maybe you can move this into its own component already.